eng-wasm/src/crypto_wasm.rs

```diff
@@ -0,0 +1,17 @@
+use super::*;
+
+const SYMMETRIC_KEY_SIZE: usize = 32;
```

Comment:
Can these constants and the constants in symmetric.rs have the same single origin and not be copied? Duplicatin may cause big problems in the future, if only one of the places will be changed.

Reply:
Yes, but where would you put them?

Reply:
I think that maybe getting `SYMMETRIC_KEY_SIZE` and `IV` should be also part of runtime services? What do you think?